spec.yaml

```diff
@@ -89,10 +89,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
-        default:
-          description: Error
+        "401":
+          description: Permission Denied
```

Comment:
To add to that, "default" is a way of communicating "there might be other response (error) types not described here".